rust/migration/src/m20220101_000001_create_blob_tables.rs

```diff
@@ -22,15 +22,6 @@ impl MigrationTrait for Migration {
             )
             .await?;
 
-        // Insert basic backing store since it must always exist.
```

Comment:
The filename is generated but the contents aren't. Changing the file is rewriting history and generally frowned upon. Not rewriting history is much more challenging however and since the only living database is my personal one the impact is very low. 

I won't be able to do this anymore once we deploy the first real instance of the rsc.

aside: The `entity` models _are_ generated and they derive their values from the state of the database at any given point in time.